code/game/machinery/computer/card.dm

```diff
@@ -66,6 +66,10 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 
 /obj/machinery/computer/card/attackby(obj/I, mob/user, params)
 	if(isidcard(I))
+		var/obj/item/card/id/insert_card = I
+		if(!insert_card.electric)
+			to_chat(user, to_chat(user, "<span class='warning'>There's no circuitry to change!</span>"))
+			return
```

Comment:
you shouldn't need this, id_insert calls try_insert, right?

Reply:
well it doesnt hurt to have a fallback

Reply:
the id_insert text is objectively better, "There's no circuitry to change" is a lot more vague. So it should use that if it is supposed to.